src/pudl/extract/eia860.py

```diff
@@ -77,6 +77,8 @@ def get_dtypes(page, **partition):
     "raw_eia860__generator_existing",
     "raw_eia860__generator_proposed",
     "raw_eia860__generator_retired",
+    "raw_eia860__generator_solar_existing",
+    "raw_eia860__generator_solar_retired",
```

Comment:
Two quick things here: 
1) What is the reasoning behind including `generator` in the table name?
2) In the raw csv, the tab for the retired table says "retired or canceled" -- this was the same for the storage table, so I gave it the suffix `_retired_canceled`. It's a little clunky, but I think it's more informative. Whatever we choose it should probably be the same for both tables.

Reply:
I used `generator` because there are generator IDs. They are just a special solar case of generators. I could be convinced that we should do that but i personally think it makes sense to make it explicit and consistent with the other generator tabs.

And I used `retired` only because that is what we use for the standard retired/canceled generators. I'm slightly partial to this shorter version partially bc shorter but also just because i think we should be consistent and if we add both suffixes it would be good to add them to the current generators tabs as well.

Reply:
Ah ok that makes sense! Is it duplicate data from what is in the generator table? If it's unique, maybe we could rename the generator table to be more specific too. Like generator_steam_existing or something like that...idk.

I also hate the clunkiness of `retired_canceled` but to your point in the generators table, it is more explicit which I think is important. Feels like small beans though. Maybe let's get a third opinion @e-belfer what do you think?

Reply:
From my understanding the current generators includes all of the generators and these two new solar and storage tables are special subsets of the generators that have specific solar and storage attributes. Because of that, I personally do not think it makes sense to rename the current generators tabs.

yea i don't disagree w/ `retired_canceled`. i just don't think its a big deal bc its entirely internal and if we name this one `retired_canceled` we should definitely also rename the existing ones which will be fine but slightly more involved.